src/component/player/player.vue

```diff
@@ -427,6 +432,16 @@
 					if (fight.data) {
 						this.getLogs()
 						this.game.init(fight)
+
+						const totalActions = fight.data.actions.length
+						for (let action = 0; action < totalActions; action++) {
+							const resurrect = fight.data.actions[action][0] == ActionType.RESURRECTION
+							if (fight.data.actions[action][0] == ActionType.PLAYER_DEAD || resurrect) {
```

Comment:
`fight.data.actions[action][0] == ActionType.PLAYER_DEAD` Could use a variable, as `resurrect` at the line before

Reply:
Also, boolean should start with "is, was, should, ...", but that depends on the coding rules :)

Reply:
done